docs/content/getting-started/install.mdx

```diff
@@ -19,6 +19,12 @@ pip install dagster dagit
 
 This will install the latest stable version of the core Dagster packages in your current Python environment.
 
+**Note for Macs with an M1 or M2 chip**: Some users have reported installation problems due to missing wheels for arm64 Macs when installing the `grpcio` package. To install `dagster` using a pre-built wheel of the `grpcio` package for M1 and M2 machines, run:
+
+```bash
+pip install dagster dagit --find-links=https://github.com/dagster-io/build-grpcio/wiki/Wheels
```

Comment:
How does one encode this --find-links shenanigan in a setup.py or requirements.txt file? e.g. if someone downloads an example with a dagster dep, how do we make sure it gets installed with the proper wheels?

Reply:
@schrockn it would be a CLI flag that you supply to the pip install in that case as well. There's not a way to encode it in the setup.py that we're aware of, doing that was Plan A but pypi didn't like it

Reply:
Unfortunately PyPI seems to actively discourage/prevent silently using a wheel outside of PyPI. They used to have [dependency_links](https://setuptools.pypa.io/en/latest/deprecated/dependency_links.html) which did exactly this but has been removed in recent versions. Our attempt at just pointing directly to our wheel in setup.py also failed to publish.
I now assume their thinking is if you are using a non standard wheel repo, it should be specified by the user via --find-links or --extra-index-url.

Reply:
`setup.py` is for "abstract" dependencies and so it only supports version specifiers to my knowledge. `requirements.txt` on the other hand allows you to point to a specific "physical" package (the full spec is [requirement specifiers](https://pip.pypa.io/en/stable/reference/requirement-specifiers/#requirement-specifiers)), so you could do e.g. this:

```
### requirements.txt

grpcio @ https://github.com/dagster-io/build-grpcio/raw/main/wheels/grpcio-1.47.2-cp39-cp39-macosx_11_0_arm64.whl
```

Reply:
@smackesey do you know if we can publish packages with `requirements.txt` like above to PyPI? `setup.py` supports [direct references](https://setuptools.pypa.io/en/latest/userguide/dependency_management.html#direct-url-dependencies) as well but PyPI [does not accept packages](https://github.com/dagster-io/dagster/pull/11760) that use them.

We want to use our wheel for m1 macs only and use the default for all other machines (intel macs, non macs). We also want a way to fallback to the old behavior (ie not use our wheel on m1s) if users already have `grpcio` installed from elsewhere. Doing all of these in `requirements.txt` seems hard.